src/main/java/mybatis/mapper/SanPhamMapper.java

```diff
@@ -214,4 +214,10 @@ public int updateSP_Status(
             @Param("status") int status,
             @Param("maSanPham") String maSanPham
     );
+
+    //Sửa thông tin của sản phẩm (bên admin)
+    final String UPDATE_SP_INFO_ADMIN = "UPDATE `san_pham` SET `ten_san_pham`=#{tenSanPham},`mo_ta`=#{moTa},`gia`=#{gia}, " +
+         "`status`=#{status},`ma_loai_san_pham`=#{maLoaiSanPham},`so_luong`=#{soLuong},`ngay_dang`=#{ngayDang} WHERE `ma_san_pham` = #{maSanPham};";
+    @Update(UPDATE_SP_INFO_ADMIN)
+    public int updateSP_info_admin(SanPham sanpham);
```

Comment:
Cái này đang làm lở dở nhan ô

Reply:
update sản phẩm info admin là sao, ai xài cái này

Reply:
sao ông ko rep zô đây luôn, admin nó chỉnh status thôi chứ nó chỉnh đống thông tin còn lại làm gì

Reply:
ờ nó hiện ở dưới
ok vậy chỉ cho khách hàng sửa chi tiết sản phẩm

Reply:
WHERE `ma_san_pham` = #{maSanPham}  AND `ma_khach_hang`= #{maKhachHang}
 tui thêm mã khách hàng để tránh sản phẩm của khách hàng khác bị sửa nhầm

Reply:
#{maKhachHang} là mã người dùng trong session